tests/test_core.py

```diff
@@ -6012,20 +5989,9 @@ def count_relocations():
       num_relocs = relocs.count('\n')
       return relocs, num_relocs
```

Comment:
Huh.. this function looks completely unused both before and after this change...

Reply:
Good eye, removed.